src/town_gui.cpp

```diff
@@ -337,13 +337,15 @@ struct TownViewWindow : Window {
 		SetDParam(1, this->town->cache.num_houses);
 		DrawString(r.left + WD_FRAMERECT_LEFT, r.right - WD_FRAMERECT_LEFT, y, STR_TOWN_VIEW_POPULATION_HOUSES);
 
-		SetDParam(0, this->town->supplied[CT_PASSENGERS].old_act);
-		SetDParam(1, this->town->supplied[CT_PASSENGERS].old_max);
-		DrawString(r.left + WD_FRAMERECT_LEFT, r.right - WD_FRAMERECT_LEFT, y += FONT_HEIGHT_NORMAL, STR_TOWN_VIEW_PASSENGERS_LAST_MONTH_MAX);
-
-		SetDParam(0, this->town->supplied[CT_MAIL].old_act);
-		SetDParam(1, this->town->supplied[CT_MAIL].old_max);
-		DrawString(r.left + WD_FRAMERECT_LEFT, r.right - WD_FRAMERECT_LEFT, y += FONT_HEIGHT_NORMAL, STR_TOWN_VIEW_MAIL_LAST_MONTH_MAX);
+		SetDParam(0, 1 << CT_PASSENGERS);
```

Comment:
why the bitshift? presumably something relating to the CARGO_LIST type, but is confusing on initial inspection

Reply:
{CARGO_LIST} takes a bitmap of cargo types. It's used to display a list of cargo types with just one parameter. I could use SetBit() but it would need a variable to operate on.